editor/components/table-of-contents/style.scss

```diff
@@ -1,8 +1,12 @@
 .table-of-contents__popover.components-popover:not(.is-mobile) .components-popover__content {
 	min-width: 380px;
+	height: 100%;
```

Comment:
I think we should just set a fix max-height IMO. What do you think @jasmussen

Reply:
You're right @youknowriad

Reply:
Actually I meant a fixed value like: `max-height: 400px` or something. Seems simpler and sufficient to me.